src/examples/zkapps/voting/preconditions.ts

```diff
@@ -0,0 +1,29 @@
+import { UInt32, UInt64 } from 'snarkyjs';
+
+export class ElectionPreconditions {
+  startElection: UInt32;
+  endElection: UInt32;
+
+  static get default(): ElectionPreconditions {
```

Comment:
Did we remove the default preconditions method? Is that version in a different PR?

Reply:
Not sure, I think can't remember removing it